src/Goerli-DssSpell.t.sol

```diff
@@ -2482,13 +2456,16 @@ contract DssSpellTest is DSTest, DSMath {
         }
     }
 
-    function test_auth() public {
-        checkAuth(false);
-    }
-
-    function test_auth_in_sources() public {
-        checkAuth(true);
-    }
+    // TODO: Fix Changelog Wards
+    // Log("Bad auth", @0xdA0C0de01d90A5933692Edf03c7cE946C7c50445, "CHANGELOG")
```

Comment:
Also this EOA has wards on the chainlog: `0x92723e0bF280942B98bf2d1e832Bde9A3Bd2F2c2`